pyswarms/single/global_best.py

```diff
@@ -129,6 +134,7 @@ def __init__(
             velocity_clamp=velocity_clamp,
             center=center,
             ftol=ftol,
+            ftol_iter=ftol_iter,
```

Comment:
Nit: similar to the one before, please add spaces between the operator:

```python
ftol_iter = ftol_iter
```

Reply:
This seems to be consistent with the above lines.
Do you recommend changing all the lines? Or is this fine?

Same query regarding the next change.

Reply:
Oof you’re right! Ha! I didnt notice that! 
Ideally there should be some spacing in between (or to be really sure, what the black formatter gives), but for this PR let’s leave it at that. No action needed for this snippet